xbmc/music/dialogs/GUIDialogSongInfo.cpp

```diff
@@ -185,6 +186,14 @@ bool CGUIDialogSongInfo::OnMessage(CGUIMessage& message)
           return true;
         }
       }
+      else if (iControl == CONTROL_BTN_PLAY)
+      {
+        const CFileItemPtr pItem = std::make_shared<CFileItem>(*m_song);
+
+        OnPlaySong(pItem);
+        return true;
+      }
+      return false;
```

Comment:
if i see it correct we will never reach the `return CGUIDialog::OnMessage(message);` below, since we always return false; ... behavior is changed. is this intended?

Reply:
Behaviour is not changed.  If the switch statement fails, we `return CGUIDialog::OnMessage(message);` - That's existing code.  False is only returned here if `GUI_MSG_CLICKED` and we don't know how to handle that (shouldn't happen really!).

Reply:
ok, thanks. then i got it wrong 😉